config/routes.rb

```diff
@@ -1,6 +1,13 @@
 Rails.application.routes.draw do
-  get 'pages/hello'
-  root 'pages#hello'
+  get 'users/index'
+  get 'users/show'
+  get 'posts/index'
+  get 'posts/show'
+  root 'users#index' 
+  resources :users do 
+    resources :posts 
+ end
```

Comment:
- [ ] We can rewrite our routes in a more RESTFUL way to achieve the desired URLs mentioned in the project requirements. When I run `rails routes` the result of the URLs I get do not match the ones provided in the project requirements. See below:

URLs from project requirements | URLs from project
:---:|:---:
![](https://user-images.githubusercontent.com/45002654/149008687-e33ca00d-89df-490d-b203-601bc363d73f.png) | ![Screenshot from 2022-01-12 20-56-57](https://user-images.githubusercontent.com/22434660/149171333-b829d880-f378-4e6f-995b-37ed05b6ee68.png)

Please consider rewriting the routes as below:

```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```